corehq/apps/callcenter/tests/test_location_owners.py

```diff
@@ -55,9 +55,14 @@ def setUpClass(cls):
     def tearDownClass(cls):
         cls.user.delete()
         cls.domain.delete()
+        cls.grandchild_location.delete()
+        cls.child_location.delete()
+        cls.root_location.delete()
+        cls.location_type.delete()
```

Comment:
This class should be calling `super().tearDownClass` and the setup should be calling the super as well.

If that's the case, I think this would be getting cleaned up properly. If they still aren't cleaned up, we have a  helper `delete_all_locations` that can be used